Documentation/git-filter-repo.txt

```diff
@@ -1039,6 +1039,15 @@ def name_callback(name):
   return name.replace(b"Wiliam", b"William")
 --------------------------------------------------
 
+The above `--name-callback` incanatation is shorthand for this raw `--commit-callback`
```

Comment:
One think I noticed is that this message happens before `--commit-callback` is explained below. I think this is ok, since it's referenced above initially, so you'll know where to look.